CameraInspector/FaceDetector.cpp

```diff
@@ -0,0 +1,14 @@
+#include "FaceDetector.h"
+
+namespace CameraInspector
+{
+
+FaceDetector::FaceDetector()
```

Comment:
It was left for Polyna's purposes
Fixed: https://github.com/LvCPP/SecureBrowser/commit/2e4ec15659a4dba11b7e27074a61821450e4ece7